bindings/go/evmc/evmc.go

```diff
@@ -0,0 +1,258 @@
+// EVMC: Ethereum Client-VM Connector API.
+// Copyright 2018 Pawel Bylica.
+// Licensed under the MIT License. See the LICENSE file.
+
+package evmc
+
+/*
+#cgo CFLAGS:  -I${SRCDIR}/../../../include
+#cgo LDFLAGS: -ldl
+
+#include <evmc/evmc.h>
+#include <evmc/helpers.h>
+#include <evmc/loader.h>
+
+#include <stdlib.h>
+#include <string.h>
+
+inline int set_option(struct evmc_instance* instance, _GoString_ name, _GoString_ value)
+{
+	if (instance->set_option)
+		return instance->set_option(instance, name.p, value.p);
+	return -1;
```

Comment:
Note: the `evmc_set_option` wrapper returns 0 in this case. Which one is preferred?

Reply:
Here it returns 1, 0 or -1. This is not super needed, but I have different warnings for 0 and -1.